ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -31,45 +31,38 @@ const {
 const DirectionInput = require('../../component/location-new/geo-components/direction.js')
 const { Direction } = require('../../component/location-new/utils/dms-utils.js')
 
-const minimumDifference = 0.0001
-
 const BoundingBoxLatLonDd = props => {
-  const {
-    north,
-    east,
-    south,
-    west,
-    setState,
-    mapEast,
-    mapWest,
-    mapSouth,
-    mapNorth,
-  } = props
+  const { north, east, south, west, setState } = props
   const [ddError, setDdError] = useState(initialErrorStateWithDefault)
-  const westMax = parseFloat(mapEast) - minimumDifference
-  const eastMin = parseFloat(mapWest) + minimumDifference
-  const northMin = parseFloat(mapSouth) + minimumDifference
-  const southMax = parseFloat(mapNorth) - minimumDifference
 
   useEffect(
     () => {
       if (props.drawing) {
         setDdError(initialErrorStateWithDefault)
       }
+      if (!ddError.error) {
```

Comment:
why do we only want to call the `validateGeo` function when there is no error? if there was some other error, like you entered a value too large, and it defaulted to 90, but your north was still less than your south, we'd want to show the north < south error, right?

Reply:
So in that case you laid out, it won't change to the north less than south error until onBlur. But this is to protect against clearing out other errors when north is _not_ less than south. Since the same error state is being used for all errors, we don't want to clear out errors when north is greater than south yet there may be some other error, like invalid UTM/UPS or latitude cannot be empty

Reply:
ohh i see okay, so we want all other errors to take precedence basically. fine by me